### PR TITLE
add travis CI for testing

### PR DESCRIPTION
**Pull request by tshedor**
Don't merge yet - I'll be testing on this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Meet Fae
 
-![Shippable Status](https://img.shields.io/shippable/5522ec125ab6cc1352b9bb77.svg)
+[![Build Status](https://travis-ci.org/wearefine/fae.svg?branch=master)](https://travis-ci.org/wearefine/fae)
 [![Code Climate](https://codeclimate.com/github/wearefine/fae/badges/gpa.svg)](https://codeclimate.com/github/wearefine/fae)
+![Shippable Status](https://img.shields.io/shippable/5522ec125ab6cc1352b9bb77.svg)
```
Comment (jamesmk) [inline README.md]: Shippable taking the backseat

**Comment by jamesmk** [comment]
@tshedor: Judge uses multiple gemfiles via `appraisal`, here's it's travis file: https://github.com/joecorcoran/judge/blob/master/.travis.yml

**Comment by tshedor** [comment]
@jamesmk I got this to pass. I had to edit one of the migrations in the dummy that was throwing on adding duplicate indexes (I had to reset my DB to get to the heart of this foreign key issue and when I `rake db:migrate`d the first time, I encountered the error).

**Comment by jamesmk** [comment]
:white_check_mark:

**Pull request merged**